shader/binning.wgsl

```diff
@@ -127,7 +127,11 @@ fn main(
         sh_count[i][local_id.x] = element_count_packed;
     }
     // element_count is the number of draw objects covering this thread's bin
-    let chunk_offset = atomicAdd(&bump.binning, element_count);
+    var chunk_offset = atomicAdd(&bump.binning, element_count);
+    if chunk_offset > bump.binning_size {
```

Comment:
I think this needs to be `chunk_offset + element_count`. What you get back is the previous value.

Reply:
Oops, yes. I think I had that in there originally and removed it for some reason. I’ll adjust them all.